feedhenry/testing-cloud-app
# RHMAP-7802 Update of dependencies for node 4.4.3

## Motivation

https://issues.jboss.org/browse/RHMAP-7802
## How to verify

**Prerequisites: mongoDB installed locally (port 27017), redis-server running locally (port 6379)**

Just try to execute `npm install` with node versions `0.10.30` and `4.4.3` and run `grunt serve` for each node version.
Application should be running on `http://localhost:8001/test`
